boefjes/boefjes/job_handler.py

```diff
@@ -222,6 +222,23 @@ def handle(self, normalizer_meta: NormalizerMeta) -> None:
                     )
                 )
 
+            if (
+                normalizer_meta.raw_data.boefje_meta.input_ooi  # No input OOI means no deletion propagation
+                and not results.observations + results.declarations + results.affirmations
```

Comment:
Nice find. I would like to suggest this:
```suggestion
                and not any((results.observations, results.declarations, results.affirmations))
```
The current implementation creates a new list just to check whether it's truthy. This suggestion is slightly more efficient since it doesn't need to create a mutable list. (Tuples are also immutable, making them a better choice when you don't need to modify the data.)

Reply:
If you care about the efficieny, creating a tuple is also not necessary:

```suggestion
                and not (results.observations or results.declarations or results.affirmations)
```

Reply:
> ```suggestion
> 
>                 and not (results.observations or results.declarations or results.affirmations)
> 
> ```


This is indeed the better solution. I was too focused on avoiding unnecessary iteration over sequences, but in this case it’s not even needed for checking empty lists

Reply:
:+1: